docs/index.md

```diff
@@ -5,3 +5,6 @@ Click below to install the Lockbox extension:
 [install][install-link]{: .button-link }
 
 [install-link]: https://testpilot.firefox.com/files/lockbox@mozilla.com/latest
+
+Note: This is a rapidly evolving prototype that will change. Any data stored is
```

Comment:
I think this should also be **strong** (as it is everywhere else), but not willing to hold up the release for it.

Reply:
Good point, I love consistency...

![screen shot 2017-11-01 at 11 18 42 am](https://user-images.githubusercontent.com/49511/32287440-7a82d2d2-bef6-11e7-92dc-42fa771eb661.png)